src/vs/workbench/api/common/menusExtensionPoint.ts

```diff
@@ -47,16 +47,15 @@ namespace schema {
 			case 'explorer/context': return MenuId.ExplorerContext;
 			case 'editor/title/context': return MenuId.EditorTitleContext;
 			case 'debug/callstack/context': return MenuId.DebugCallStackContext;
-			case 'debug/toolbar': return MenuId.DebugToolBar;
 			case 'debug/toolBar': return MenuId.DebugToolBar;
```

Comment:
API breakage?

Reply:
Yeah we had both for backwards compatibility, even though debug/toolbar was "deprecated". I am not sure if there are extensions which really on this.

Reply:
High risk. You could know relatively easy (telemetry, search extension usages) or just keep it given the low price of supporting it. btw: deprecated doesn't mean that we will remove it

Reply:
I just searched top 1000 extensions usages on `debug/toolbar` usage and only one extension used the deprecated name - the extension is not that popular and I have created an issue for then.
So I suggest the following: let's go with the removal, and if needed we can easily reintroduce it as part of a recovery build if somebody complains.
fyi @weinand

Reply:
Great catch @jrieken, I really thought this was an oversight and failed to see the case difference. I'll leave it up to @isidorn and @weinand. Sorry all!